stores/gaugeStore.js

```diff
@@ -1024,9 +1028,9 @@ class Store {
   }
 }
 
-function userRemainingStake(balance, totalBalance, veTokenBalance, totalVeTokenSupply) {
+function userRemainingStake(balance, totalBalance, veTokenBalance, totalVeTokenSupply, boostValue) {
   let currentStake = balance;
```

Comment:
`currentStake` and `gaugeBoostValue` vars seems unneeded you can directly use the parameters